CHANGELOG.md

```diff
@@ -78,7 +78,8 @@
 - Fixed AssertionError in `Series.drop_duplicates` and `DataFrame.drop_duplicates` when called after `sort_values`.
 - Fixed a bug in `Index.to_frame` where the result frame's column name may be wrong where name is unspecified.  
 - Fixed a bug where some Index docstrings are ignored. 
-
+- Fixed a bug in `Series.reset_index(drop=True)` where the result name may be wrong.
```

Comment:
I this is bit broad. Maybe consider changing to `Fixed a bug in ... where index column name in result maybe wrong.

Reply:
It's not the index column name, It return a series so the result name mean the series name was wrong.